docs/modules/ROOT/pages/upgrades.adoc

```diff
@@ -0,0 +1,57 @@
+:contract_class: https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/contract-classes/[Contract Class]
+:class_hash: https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/class-hash/[class hash]
+:replace_class_syscall: https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/system-calls-cairo1/#replace_class[replace_class]
+
+= Upgradeability
+
```

Comment:
Should we include a small section on the IUpgradeable interface? It might be useful to state its purpose for interoperability and plugins as mentioned here https://github.com/OpenZeppelin/cairo-contracts/pull/603#pullrequestreview-1406490206

Reply:
Also, I think it'd be helpful to have a how-to guide. Maybe just a `Usage` section might suffice? WDYT?

Reply:
Done! Let me know what you think.